### ISSUE TITLE
Release source code?

### ISSUE THREAD

**Issue by ggilley** [xellows1305/Document-Image-Dewarping#20]
The results look very interesting. Releasing the source code would help with reproducing the results. Thanks!

**Comment by zacharywhitley** [comment]
It's been 7 years. Just checkin in to see if you might ever publish the code for this repo? Thanks

**Comment by vincentaxhe** [comment]
likely never, the smart brain is willing to buried with it.

**Comment by zacharywhitley** [comment]
That’s unfortunate. Thank you for the reply.

**Comment by SpicyCatGames** [comment]
Yeah, this is what happens in academia. Nothing ever gets used in the real world in computer science these days.

**Comment by snowfluke** [comment]
Anybody tried to email the author?

**Comment by zacharywhitley** [comment]
Is their email in the paper? Give it a try.

**Comment by snowfluke** [comment]
Nope the email wasn't on the paper.

Close I can get is his Google Scholar profile:

https://scholar.google.com/citations?user=cV4h5MsAAAAJ&hl=ko

**Comment by lloydjatkinson** [comment]
The author seems to make a habit of doing this. They have another repo with results and data only but no code: https://github.com/taeho-kil/Scene-Text-Rectification

This seems ethically questionable and I can't tell if it's maliciousness or incompetence.

**Comment by zacharywhitley** [comment]
I don't think there's any need to attack the author or ascribe intentions. They don't have any obligation to release their code and they have contributed in the form of a paper describing their work. I agree that it's frustrating to see this site and expect that the code might be released but there are many things that could have contributed to this that are not malicious or incompetent. I hope the author sees this more as interest in their work and that their work can contribute beyond just the paper. Otherwise, I'm afraid it will simply go on the pile of papers and fade away.
